Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -140,6 +140,11 @@ open class NavigationMapView: UIView {
      */
     private(set) var predictiveCacheManager: PredictiveCacheManager?
     
+    /**
+     A `zoomLevelToShowManeuverArrow` used to decide which zoom level to show the maneuver arrow. When the zoom level is smaller than the value, the arrow will hide. when zoom level is larger than the value, the arrow will show up. Defaults to `13.0`.
+     */
+    public var zoomLevelToShowManeuverArrow: Double = 13.0
```

Comment:
I'm not sure that we should introduce new public property, which allows to control threshold zoom level value for maneuver arrow. Looks like based on `1.x` implementation we're not using similar value for `minimumZoomLevel` property in `2.0`. For example:

https://github.com/mapbox/mapbox-navigation-ios/blob/cf946f822afd06c0dc7fae6923eef787dd871465/Sources/MapboxNavigation/NavigationMapView.swift#L870

In `2.0` we're using:

https://github.com/mapbox/mapbox-navigation-ios/blob/7637a8d9b0419e4b7cba9a3e813441286beb7840/Sources/MapboxNavigation/NavigationMapView.swift#L705

Can you please use similar value in `2.0`? I think it should fix original issue without adding new logic and public variable.

Reply:
Thanks for the suggestion! I'll fix this part and remove the introduced public property.